go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/getlantern/lantern-client
 
-go 1.22.3
-
-toolchain go1.22.8
+go 1.23
 
 // replace github.com/getlantern/flashlight/v7 => ../flashlight
+
 // replace github.com/getlantern/ipproxy => ../ipproxy
 // replace github.com/getlantern/fronted => ../fronted
 // replace github.com/getlantern/pathdb => ../pathDb/pathDb
 
+replace github.com/getlantern/flashlight/v7 => github.com/getlantern/flashlight/v7 v7.6.140-0.20241119222126-ff487ae5ee73
```

Comment:
We definitely don't want this.  This will make future attempts to update flashlight fail in surprising ways.  Could you update flashlight the normal way (remove this replace directive and just run `go get`)?